infection_monkey/network/network_scanner.py

```diff
@@ -36,8 +36,26 @@ def initialize(self):
         self._ranges = [NetworkRange.get_range_obj(address_str=x) for x in WormConfiguration.subnet_scan_list]
         if WormConfiguration.local_network_scan:
             self._ranges += get_interfaces_ranges()
+        self._ranges += self._get_inaccessible_subnets_ips()
         LOG.info("Base local networks to scan are: %r", self._ranges)
 
+    def _get_inaccessible_subnets_ips(self):
+        subnets_to_scan = []
```

Comment:
I'm going to need some documentation on what's going on here.

Reply:
fixed